CMakeLists.txt

```diff
@@ -18,6 +18,14 @@ if(CMAKE_CXX_BYTE_ORDER STREQUAL "BIG_ENDIAN")
   add_definitions("-DOUTRAGE_BIG_ENDIAN")
 endif()
 
+
+# 64 bit machines have a different game checksum than 32 bit machines
```

Comment:
IMO the CHECKSUM value was previously only in the scripts module because they're the ones that get the value blindly where the runtime computes it. not totally for or against moving either way.

Reply:
I moved it to level up, so both script and Descent will have access for define. Logically Osiris_CreateGameChecksum() belongs to Descent, while script only consumes CHECKSUM.

This is not ideal solution, and I planning sometime just move some defines to generated by cmake header file.